src/lib/rest/rest.cpp

```diff
@@ -1041,6 +1042,16 @@ static int connectionTreat
       ciP->httpHeaders.servicePath = defaultServicePath(url, method);
     }
 
+    /* X-Forwared-For (used by a potential proxy on top of Orion) overrides ip */
+    if (ciP->httpHeaders.xforwardedFor == "")
+    {
+      LM_TRANSACTION_SET_FROM(ip);
```

Comment:
Not so much... just 5 functions as much. Very easy to change with "Replace All" ;)